docs/rules/no-duplicate-imports.md

```diff
@@ -14,57 +16,47 @@ import { find } from 'module';
 
 This inspection reports any duplicated module in an import statement.
 
-The following patterns are considered problems:
+Example of **incorrect** code for this rule:
 
 ```js
 /*eslint no-duplicate-imports: "error"*/
 
 import { merge } from 'module';
-import path from 'another-module';
-import { find } from 'module';
-
-
-import { merge } from 'module';
+import something from 'another-module';
 import _, { find } from 'module';
 ```
 
-The following patterns are not considered problems:
+Example of **correct** code for this rule:
 
 ```js
 /*eslint no-duplicate-imports: "error"*/
 
-import { merge, find } from 'module';
-import path from 'another-module';
+import _, { merge, find } from 'module';
+import something from 'another-module';
 ```
 
 ## Options
 
 This rule takes one optional argument, an object with a single key, `includeExports` which is a `boolean`. It defaults to `false`.
 
-With this option set to `true`, the following patterns are considered problems:
+If re-exporting from an imported module, you should add the imports to the `import`-statement, and export that directly, not use `export ... from`.
+
+With this option set to `true`, the following examples is **incorrect** code for this rule:
 
 ```js
 /*eslint no-duplicate-imports: ["error", { includeExports: true }]*/
 
 import { merge } from 'module';
-import path from 'another-module';
 
 export { find } from 'module';
-
-
-import _ from 'module';
-const find = _.find;
-
-export { find as lodashFind } from 'module';
 ```
 
-The following patterns are not considered problems:
+Example of **correct** code for this rule:
```

Comment:
When an example relates to a specific option, we usually include the option at the end of the caption like so:

```
Example of **correct** code for this rule with the `{ includeExports: true }` option:
```

I wouldn't worry about implementing that change just yet as there's some planned changes to the text patterns, amongst other stuff. As long as they are using the consistent "Example of **(in)correct** code for this rule ...:" pattern they will ensure code blocks are colorized and iconified which massively improves readability.